templates/settings.py.erb

```diff
@@ -1,3 +1,16 @@
+################################################################################
+# !!! WARNING: DO NOT EDIT THIS FILE !!!
+################################################################################
+# File managed by Puppet.
+# Module: <%= scope['module_name'] %>
+################################################################################
+# Not only are edits likely to be overwritten, there is also a strong
```

Comment:
They're not just likely, I think we can safely say they will be overwritten at some point.